src/main/java/hillelJavaEE_02/doctor/DoctorConfig.java

```diff
@@ -0,0 +1,21 @@
+package hillelJavaEE_02.doctor;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+@Configuration
+@ConfigurationProperties(prefix = "doctor-specialization")
+public class DoctorConfig {
+    private List<String> list;
+
+    DoctorConfig() {
+        this.list = new CopyOnWriteArrayList<>();

```

Comment:
:)
there is no need in thread safe collection. spring initializes all entities in single thread